docs/docker/intro.md

```diff
@@ -1,14 +1,14 @@
 ## Docker 🐳
 
-Docker is a container-based virtualization software that allows users to easily create a development environment without installing any virtualization software (e.g. VirtualBox).
+Docker is a virtualization software that run software named **container**. A container is an isolated process containing your application which can communicate with other application. This software allow you to create multiple containers easily
```

Comment:
We need to carefuly choose which nouns to use, and define them if necessary.
Here, **containers** are defined both as softwares and isolated processes. Thus, the last sentence could refer both to Docker or containers.
I find this definition quite clear : https://www.webopedia.com/TERM/D/docker.html

What do you think about this @MarcInthaamnouay :

> Docker is a software that allows [Operating system level virtualization](https://en.wikipedia.org/wiki/Container_(virtualization)) thanks to user space instances called **containers**. These containers are isolated processes containing applications that can communicate with each other and with other applications outside of their container thanks to Docker networking.

Reply:
Indeed we could use this definition